libretro_core_options.h

```diff
@@ -253,7 +253,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(mdec_yuv),
       "MDEC YUV Chroma Filter",
       NULL,
-      "Improves the quality of FMV playback by reducing 'macroblocking' artifacts (squares/jagged edges). Only supported by the Vulkan renderer.",
+      "Improve the quality of FMV playback by reducing 'macroblocking' artifacts (squares/jagged edges). Only supported by the Vulkan renderer.",
```

Comment:
I don't think the word "macroblocking" needs quotes. Maybe it was quoted to make it more visible for non-expert users, but isn't there another way to do that?

Reply:
Who cares, the quotes don't belong either way.